.github/workflows/ci_cd.yml

```diff
@@ -102,7 +102,7 @@ jobs:
 
       - name: Unit Testing
         run: xvfb-run coverage run -m --source=pytest_pyvista --module pytest --verbose .
-      - uses: codecov/codecov-action@v4
+      - uses: codecov/codecov-action@v3
```

Comment:
```suggestion
      - uses: codecov/codecov-action@v4
```

Reply:
I think it was deprecated.

Reply:
See https://github.com/pyvista/pytest-pyvista/actions/runs/6254763049/job/16982901626